src/main.ts

```diff
@@ -0,0 +1,16 @@
+export { BooleanSBVR }     from './types/BooleanSBVR'
```

Comment:
Why did you add an `SBVR` suffix to all of the files/types?

Reply:
Some of the types conflict with existing ts-types (File,JSON,ecc).